src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/FileStorageService.cs

```diff
@@ -160,31 +157,38 @@ public async Task DeletePublicBlob(string path)
             await _publicBlobStorageService.DeleteBlob(PublicContentContainerName, path);
         }
 
-        public async Task<IEnumerable<FileInfo>> ListPublicFiles(string publication, string release)
+        [Obsolete("Use GetPublicFileInfo with a file reference instead of filename")]
+        public async Task<FileInfo> GetPublicFileInfo(ReleaseFileTypes type, string path)
```

Comment:
As below, would suggest just removing these methods in favour of `GetBlob` and `CheckBlobExists` (these are ultimately more re-usable).

Reply:
I've done that now